atc/metric/emitter/prometheus.go

```diff
@@ -151,6 +154,22 @@ func (config *PrometheusConfig) NewEmitter() (metric.Emitter, error) {
 	})
 	prometheus.MustRegister(buildsRunning)
 
+	concurrentRequestsLimitHit := prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "concourse",
+		Subsystem: "builds",
+		Name:      "started_total",
+		Help:      "Total number of Concourse builds started.",
+	})
+	prometheus.MustRegister(concurrentRequestsLimitHit)
+
+	concurrentRequests := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "concourse",
+		Subsystem: "builds",
+		Name:      "running",
+		Help:      "Number of Concourse builds currently running.",
```

Comment:
also probably a copy-paste

Reply:
total copy & paste :grin:  there were other issues, so I just left for a future commit :+1:

Reply:
https://github.com/concourse/concourse/pull/5429#issuecomment-616821387